src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area, Draw {

```

Comment:
it's redundant implementation, cause Figure should already implement Area and Draw